### PR TITLE
Fix/ Gray table background color

### PR DESCRIPTION
**Pull request by obniavko**
dev
## ZeroWaste project

* [Project ticket #682](https://github.com/ita-social-projects/ZeroWaste/issues/682)

## Code reviewers

- [ ] @kisiohlova 
- [ ] @loqimean 

## Summary of issue

Gray background color in the table on the "Calculators" page.

## Summary of change

Removed gray background color.

Gray1

Gray2

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: app/views/account/calculators/index.html.erb [hunk 1]
```diff
@@ -25,7 +26,7 @@
     </thead>
     <tbody class="admin-table-links">
       <% @calculators.each do |calculator| %>
-        <tr class="<%= calculator.preferable ? "table-success" : "table-light" %>">
+        <tr>
```
Comment (kisiohlova) [inline app/views/account/calculators/index.html.erb:29]:
```suggestion
        <tr class="<%= calculator.preferable ? "table-success" : "text-inherit" %>">
```
So the preferable calculator is highlighted.

**Review by kisiohlova — CHANGES_REQUESTED** [review]

File: app/views/calculators/index.html.erb [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
         <th scope="col"><%= sort_link(@q, :id, t('.id_column'), default_order: :desc) %></th>
         <th scope="col"><%= sort_link(@q, :name, t('.calculator_name'), default_order: :desc) %></th>
-        <th scope="col"><%= t('.calculator_actions') %></th>
+        <th scope="col" class="text-center"><%= t('.calculator_actions') %></th>
       </tr>
     </thead>
     <tbody class="admin-table-links">
       <% @calculators.each do |calculator| %>
-        <tr class="<%= calculator.preferable ? 'table-success' : 'table-light' %>">
+        <tr>
```
Comment (loqimean) [inline app/views/calculators/index.html.erb:13]: те ж саме, що Настя написала

**Review by loqimean — CHANGES_REQUESTED** [review]

File: app/views/calculators/index.html.erb (hunk 2 above)
Comment (obniavko) [reply app/views/calculators/index.html.erb]: Fixed.

File: app/views/account/calculators/index.html.erb (hunk 1 above)
Comment (obniavko) [reply app/views/account/calculators/index.html.erb]: Fixed.

**Review by kisiohlova — APPROVED** [review]

**Review by loqimean — APPROVED** [review]

**Pull request merged**